training/main.py

```diff
@@ -1,14 +1,22 @@
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker  
 from sqlalchemy.ext.declarative import declarative_base
+import os
+
 
 Base = declarative_base()
 
-engine = create_engine('postgresql+psycopg2://postgres:postgres@localhost/baseparaflask')
+#engine = None
+#session = None
+#Siempre se sustituyen por None, aunque las cambie abajo
+engine = create_engine(os.environ.get("DATABASE_URL"))
 Session = sessionmaker(engine)
 session = Session()
 
+
 def init_db():
+    #global engine
+    #global session
```

Comment:
Please remove this commented code

Reply:
It will be fixed in the next PR in the next branch.